commit/chainfee/observation.go

```diff
@@ -17,19 +18,21 @@ func (p *processor) Observation(
 	if err != nil {
 		return Observation{}, err
 	}
+	sortedChains := supportedChains.ToSlice()
+	sort.Slice(sortedChains, func(i, j int) bool { return sortedChains[i] < sortedChains[j] })
```

Comment:
Some comments on why we need the input to be sorted would be great. Does it mean that the following ccipReader's methods expect input to be sorted? `GetAvailableChainsFeeComponents`, `GetWrappedNativeTokenPriceUSD`, `GetChainFeePriceUpdate`